TransferMethod/Sources/SelectTransferMethodTypePresenter.swift

```diff
@@ -290,4 +303,30 @@ final class SelectTransferMethodTypePresenter {
     func setInsights(hyperwalletInsights: HyperwalletInsightsProtocol) {
         self.hyperwalletInsights = hyperwalletInsights
     }
+
+     private func trackTransferMethodClick() {
+        let impressionParams = [
+            InsightsTags.country:
+                self.selectedCountry,
+            InsightsTags.currency:
+                self.selectedCurrency,
+            InsightsTags.transferMethodType:
+                self.selectedTransferMethodType
+        ]
+      getInsights().trackImpression(pageName: pageName, pageGroup: pageGroup, params: impressionParams)
+  }
+    private func trackCountryClick() {
+           getInsights()
+               .trackClick(pageName: pageName,
+                           pageGroup: pageGroup,
+                           link: link,
```

Comment:
Go through the requirement document again and see what's the value for the link in case Select Country and use that here instead of wrong value "select-transfer-method"

Reply:
My Bad , I was looking into the sample JSON in the document which was select-transfer-method for Link tag in Select Country and Select Currency .